SourceX/qol.cpp

```diff
@@ -160,11 +160,11 @@ void DrawXPBar(CelOutputBuffer out)
 	int curXp = ExpLvlsTbl[charLevel];
 	int prevXp = ExpLvlsTbl[charLevel - 1];
 	int prevXpDelta = curXp - prevXp;
-	int prevXpDelta_1 = plr[myplr]._pExperience - prevXp;
+	Uint64 prevXpDelta_1 = plr[myplr]._pExperience - prevXp;

```

Comment:
I'd put this assignment after the `if` guard on the line below

Reply:
Can you elaborate why? Would that be related to fixing the warnings?

Reply:
To make the code easier to understand (see what next line does)

Reply:
Placing it after the if-guard makes it clear that the code does not overflow the unsigned integer

Reply:
> To make the code easier to understand

This PR is exclusively to fix warnings however, not code readability. I'm fine with your suggestion, but it should ideally be done in a separate change (at least a separate commit).

Reply:
It's a non-trivial "warning fix only" change because an unsigned integer behaves differently from a signed integer here.